src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx

```diff
@@ -380,11 +380,15 @@ const MinimumSize = ({ partition, units }: MinimumSizePropTypes) => {
   const conversionFactor = getConversionFactor(units);
 
   const convertToDisplayUnits = (minSize: string) => {
-    return (parseInt(minSize) / conversionFactor).toString();
+    return minSize.length > 0
+      ? (parseInt(minSize) / conversionFactor).toString()
+      : '';
```

Comment:
Should there be a warning/error in a case of empty minimum size? Or we can simply default to zero as the empty value. What do you think?

Reply:
Ah, default to zero seems to be consistent with V1 behaviour so I'd go in that direction
![v1-min-size](https://github.com/osbuild/image-builder-frontend/assets/49452678/e99ab716-5717-4924-8979-6f1459da7ef2)

Reply:
not allowing empty value during editing makes for a weird behavior, when you want to delete current value and type in a new value.
We could add default to 0 on Blur, but that's still invalid value, so it should then fail validation, right? :thinking:

Reply:
Right, so I've fixed it by adding the validator :)

Reply:
Ah I see, when you add 0, it allows you to type after 0... that makes sense actually :) let me do that :+1:

Reply:
oki, done, no empty strings allowed! ;)